.circleci/config.yml

```diff
@@ -517,3 +538,20 @@ workflows:
           requires: [build]
       - distributions:
           requires: [test, container-structure]
+  manual-e2e:
+    when:
+      equal: [true, << pipeline.parameters.manual_e2e >>]
+    jobs:
+      - go_cache:
+          name: go_cache-<< pipeline.parameters.e2e_param_arch >>
+          arch: << pipeline.parameters.e2e_param_arch >>
+      - build:
+          name: build
```

Comment:
I think we should be downloading the github artifact in our e2e test as we already have everything ready right?

Reply:
Yes, that'll save some time.